python/sample-apps/function/lambda_function.py

```diff
@@ -3,6 +3,30 @@
 import aiohttp
 import asyncio
 import boto3
+import os
+import time
+
+from opentelemetry import _metrics
+from opentelemetry.exporter.otlp.proto.grpc._metric_exporter import (
+    OTLPMetricExporter,
+)
+import opentelemetry.exporter.otlp.proto.grpc.version
+
+print( "opentelemetry.exporter.otlp.proto.grpc version", opentelemetry.exporter.otlp.proto.grpc.version.__version__)
+
+from opentelemetry._metrics import (
```

Comment:
Should we use the metrics library? I know they announced GA for it, but from what I've seen OTel Python is still making updates that we could afford to wait to include?

Reply:
The whole idea of this sample is to show how to use the metrics library. I know this is in active development, but this allows users to see how the functionality would work. Not many changes are in the library in terms of API & SDK. There is a mention about some things that are broken at the minute in the first comment to give some ideas about things that may change.

Reply:
I think the question is whether this should use `metrics` or `_metrics`.  The former being the stable release version of the metrics API that should be used going forward.

Reply:
Lates Release available [1.11.1](https://github.com/open-telemetry/opentelemetry-python/releases/tag/v1.11.1) is still using ```_metrics``` - will update on a new PR once the next stable version is released